pyproject.toml

```diff
@@ -1,2 +1,7 @@
 [tool.black]
-line-length = 79
+exclude = '/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv.*|_build|buck-out|build|dist)/'
+line-length = 88
```

Comment:
Change of line-length? Any particular reason?

Reply:
Yes, it is to match what PRAW is set to.